libraries/AP_AHRS/AP_AHRS_NavEKF.h

```diff
@@ -140,6 +140,7 @@ class AP_AHRS_NavEKF : public AP_AHRS_DCM
     // return true if the estimate is valid
     bool get_relative_position_NED_home(Vector3f &vec) const override;
     bool get_relative_position_NED_origin(Vector3f &vec) const override;
+    void get_relative_position_NEU_origin_cm(Vector3f &vec) const;
```

Comment:
I agree that it isn't the best, but without it will be prone to error as all calculus on copter are made on NEU and in cm. So easy to forget it. The right direction should be to pass the inertialnav removal and then convert copter to NED.